src/coprocessor/endpoint.rs

```diff
@@ -79,22 +81,27 @@ impl Host {
         cfg: &Config,
         pool: ReadPool<ReadPoolContext>,
     ) -> Host {
+        // Use read pool's max task config
+        let max_running_task_count = pool.get_max_tasks();
```

Comment:
It confused me, what is L69 used for?

Reply:
a reminder to remove it in future, since read pool itself manages a max_running_task_count, once we totally switch to read pool, we don't need to maintain our own max_running_task_count any more.